src/packages/cli/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -39,73 +37,159 @@ type TestScenario = {
   expect: any
 }
 
-export function integrationTest(testScenarios: TestScenario[]) {
+type Context = {
+  fs: FSJetpack
+}
+
+/**
+ * Integration test keyword arguments
+ */
+type Input<Client> = {
+  /**
+   * Settings to control things like test timeout and Prisma engine version.
+   */
+  settings?: {
+    /**
+     * How long each test case should have to run to completion.
+     *
+     * @default 10_000
+     */
+    timeout?: number
+    /**
+     * The version of Prisma Engine to use.
+     *
+     * @dynamicDefault The `prisma.version` value in this package's package.json
+     */
+    engineVersion?: MaybePromise<string>
```

Comment:
Why did the postgresql integrations tests use this but not the sqlite tests?

Reply:
We should have it everywhere